internal/dag/httpproxy_processor.go

```diff
@@ -705,7 +705,8 @@ func (p *HTTPProxyProcessor) validHTTPProxies() []*contour_api_v1.HTTPProxy {
 			valid = append(valid, proxy)
 			continue
 		}
-		fqdnHTTPProxies[proxy.Spec.VirtualHost.Fqdn] = append(fqdnHTTPProxies[proxy.Spec.VirtualHost.Fqdn], proxy)
+		fqdn := strings.ToLower(proxy.Spec.VirtualHost.Fqdn)
```

Comment:
I think you'll have to do a rebase and remove the commits referencing the case-insensitive fqdn code